packages/contracts-bedrock/src/Safe/DeputyGuardianModule.sol

```diff
@@ -0,0 +1,145 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { Safe } from "safe-contracts/Safe.sol";
+import { Enum } from "safe-contracts/common/Enum.sol";
+
+import { SuperchainConfig } from "src/L1/SuperchainConfig.sol";
+import { OptimismPortal2 } from "src/L1/OptimismPortal2.sol";
+import { IDisputeGame } from "src/dispute/interfaces/IDisputeGame.sol";
+import { ISemver } from "src/universal/ISemver.sol";
+
+import "src/libraries/DisputeTypes.sol";
+
+/// @title DeputyGuardianModule
+/// @notice This module is intended to be enabled on the Security Council Safe, which will own the Guardian role in the
+///         SuperchainConfig contract. The DeputyGuardianModule should allow a Deputy Guardian to administer any of the
+///         actions that the Guardian is authorized to take. The security council can revoke the Deputy Guardian's
+///         authorization at any time by disabling this module.
+contract DeputyGuardianModule is ISemver {
+    /// @notice Error message for unauthorized access
+    error OnlyDeputyGuardian();
+
+    /// @notice Error message for failed transaction execution
+    error TransactionExecutionFailed(string);
+
+    /// @notice Emitted when the SuperchainConfig is paused
+    event Paused(string identifier);
+
+    /// @notice Emitted when the SuperchainConfig is unpaused
+    event Unpaused();
+
+    /// @notice Emitted when a DisputeGame is blacklisted
+    event DisputeGameBlacklisted(IDisputeGame game);
+
+    /// @notice Emitted when the respected game type is set
+    event RespectedGameTypeSet(GameType gameType);
+
+    /// @notice The Safe contract instance
+    Safe internal immutable SAFE;
+
+    /// @notice The SuperchainConfig's address
+    SuperchainConfig internal immutable SUPERCHAIN_CONFIG;
```

Comment:
Technically this only has to be `IPausable`. Naming it superchain config is more descriptive but less flexible. Just noting tradeoffs

Reply:
In this case I have a strong preference for SuperChainConfig, as it is the only contract with that interface in our system and the usage of this module is very specific to it.